package.json

```diff
@@ -30,8 +30,8 @@
     "contributors:generate": "all-contributors generate",
     "famadd": "all-contributors add",
     "famgen": "all-contributors generate",
-    "checkdocs": "remark ./markdown/ --quiet --frail",
-    "fixdocs": "remark ./markdown/ --quiet --frail --output"
+    "checkdocs": "remark markdown --use remark-preset-lint-consistent --use remark-preset-lint-recommended --quiet --frail",
+    "fixdocs": "remark markdown --use remark-preset-lint-consistent --use remark-preset-lint-recommended --output"
```

Comment:
Interesting. I wasn't aware of this file. 

The reason I added those flags was because without them, the command fails with 10,409 warnings. At least on my machine it seems the file is not being used.

Is there any difference for you when running the following:
```shell
npx remark markdown --quiet --frail
```

```shell
npx remark markdown --rc-path markdown/.remarkrc.yaml --quiet --frail
```

```shell
npx remark markdown --use remark-preset-lint-consistent --use remark-preset-lint-recommended --quiet --frail
```

Reply:
Did you perhaps run this prior to the addition of `remark-frontmatter`? 
Do you have it installed?

Config file is based on these docs: https://github.com/remarkjs/remark/tree/main/packages/remark-cli#example-config-files-json-yaml-js

If my reading of it is correct, it *should* get picked up.

But I'll try running the different commands you suggest and report here.

Reply:
I am on `develop` and ran `yarn kickstart`. I still see the issue. Let me put up a PR to see whether it is reproducible in a workflow.

Reply:
It failed in the workflow. See https://github.com/freesewing/freesewing/pull/1919 and https://github.com/freesewing/freesewing/runs/5264198709?check_suite_focus=true

Reply:
This remark project is strange. I can find old commits from like 5 years ago adding support for this but I can't find any of the code today.

Reply:
Don't know if you still want this but:
```
time npx remark markdown --quiet --frail
...
⚠ 10425 warnings

real	0m16,312s
user	0m19,045s
sys	0m0,511s
```

```
time npx remark markdown --rc-path markdown/.remarkrc.yaml --quiet --frail
...
⚠ 10425 warnings

real	0m14,547s
user	0m16,947s
sys	0m0,327s
```

```
time npx remark markdown --use remark-preset-lint-consistent --use remark-preset-lint-recommended --quiet --frail
...
⚠ 16 warnings

real	0m13,300s
user	0m15,859s
sys	0m0,387s
```

Reply:
I reacted a `.remarkrc` file in the project root (JSON), also doesn't get picked up

Reply:
Hey, hear me out: What if those 10.000 warnings are actually the correct result?
Like, what if it is actually picking up the config file. I think it actually does.
Because there's tons of warning like this: ``Emphasis should use `_` as a marker`` and well, that's what's configured in the config file. And it's different from the default.

So the previous run changes all emphasis to use `*` and now it's being changed back to `_`.
So the run with the 10000+ warnings is actually the one we want.